string    grasp_name
uint16     type_id
float32   scale
---
bool success
